### DISCUSSION TITLE
Context menu doesn't work with selection of several files

### DISCUSSION THREAD

**Discussion by XevianDT** [moudey/Shell#115] [General]
Hi, already was to able to add some third apps with Shell and works flawlessly, but with several files selected, items in new context menu doesn't work, just won't open. Any advice in the code?

SS in example

![2022-12-19_10-58-51](https://user-images.githubusercontent.com/121037740/208450872-1038bfc7-ace0-4fcb-b8f1-0628e187600c.png)

![2022-12-19_10-59-13](https://user-images.githubusercontent.com/121037740/208450903-c5fa8ee2-3a99-4368-8cdb-b462156a2269.png)

Thanks in advance

**Comment by moudey** [comment]
Set `mode` property to `multiple`

```
item(mode=multiple title='item 1')

item(type='file' mode=multiple title='item 1')
```
For more on the values of `mode` property of this [link](https://nilesoft.org/docs/syntax/dynamic)

**Comment by XevianDT** [comment]
Thanks, changes made and shell now open submenu, but when i select several files, opens only the highlighted one

**Comment by moudey** [comment]
Use `@sel` function to fetch all selected files

```
item(type='file' mode=multiple title='item 1' cmd='app.exe' args='@sel')
```
